lms/djangoapps/completion/static/completion/js/viewed_event.js

```diff
@@ -0,0 +1,150 @@
+/* use es5 */
```

Comment:
What does this directive do? It seems wrong to me when there are clearly ES6 classes in this file...

Reply:
I think in an earlier iteration of the file, I needed this to silence an eslint issue: IIRC, the `Set` global was unknown without it, though I can't find any documentation about it now.  Removing it doesn't seem do any harm, so I'll take it out.